src/components/Search/ListView.tsx

```diff
@@ -29,22 +29,15 @@ function TextDiv({ children }: ReactChildren) {
 }
 
 export default function List(
-  { classes, setList } : {classes: CLASS[], setList: React.Dispatch<React.SetStateAction<boolean>>},
+  { classes, user_id } : {classes: Class[], user_id: any},
 ) {
-  if (classes.length === 0) return <div>0 result</div>;
   return (
-    <div>
-      <div className="text-mint-orange mt-2">
-        {classes.length}
-        {' '}
-        results
-      </div>
-      <button type="button" className="text-center text-white text-xs font-black uppercase tracking-wide rounded-md bg-green-400 px-2 py-1 mt-2" onClick={() => setList(false)}>Map</button>
+    <div data-testid="list-container">
       <div>
         {classes.map(({
           class_id, name, time, duration, total_rating, locations,
         }) => (
-          <div key={class_id} className="flex items-start mt-4 bg-white w-full">
+          <div key={class_id} className="flex items-start mt-4 bg-white w-full" data-testid="list-item">

```

Comment:
Test using getByRole/getByText/etc if possible, data-testid as last resort

Reply:
Thanks for checking! I updated most of the getByTestId testings into getByRole and getByText, except one test call "renders markers correctly".
because in component <Map /> imported google map from @react-google-maps/api, no html element or text shown in screen. Therefore, I used data-testid as the last resort